config/brakeman.ignore

```diff
@@ -29,6 +29,9 @@
       },
       "user_input": "params[:list].presence",
       "confidence": "High",
+      "cwe_id": [
```

Comment:
Are these brakeman changes related?

Reply:
These changes are not directly related but an artefact of re-running Brakeman to update the fingerprint of our ignoring the use of `Redis.current|new`. I'm happy to remove them if they are confusing?

Reply:
Sorry for the delayed response! I'm happy for the changes to be part of this PR, I just wondered why they are present. Thanks for explaining!